markdown/en/api/ff-loading-spinner.api.md

```diff
@@ -0,0 +1,13 @@
+## `ff-loading-spinner`
+___
+### Properties
+| Name | Description |
+| ---- | ----------- |
+| **is-active**&nbsp;(Boolean) | If set, the element is displayed and animated. Otherwise the element is hidden. |
+| **stroke-color**&nbsp;(String) | Takes a CSS color value and applies it to the spinning stroke. |
+
+### Methods
+| Name | Description |
+| ---- | ----------- |
+| **show** | Activates/Shows the element. This is equal to setting the `is-active` attribute. |
```

Comment:
I find `Calling it is equal to...` would fit slightly better into the context.

Same goes for the next line.